### PR TITLE
Support the simplest form of PREPROCESSOR and POSTPROCESSOR in CREATE MODEL

### PR DESCRIPTION
**Pull request by eddyxu**

### CODE REVIEW

File: src/main/scala/ai/eto/rikai/sql/model/ModelSpec.scala [hunk 1]
```diff
@@ -38,6 +40,12 @@ class ModelSpec(
   def getOptions: java.util.Map[String, String] =
     mapAsJavaMap(options.getOrElse(Map.empty))
 
+  /** Provide access to pre-processor via py4j. It can return Null / None in python. */
```
Comment (changhiskhan) [inline src/main/scala/ai/eto/rikai/sql/model/ModelSpec.scala:43]: does Scala Option not map properly to python value?

File: src/main/scala/ai/eto/rikai/sql/spark/execution/CreateModelCommand.scala [hunk 2]
```diff
@@ -31,27 +31,32 @@ case class CreateModelCommand(
...
     table: Option[TableIdentifier],
     replace: Boolean,
     options: Map[String, String]
 ) extends ModelCommand
     with Logging {
 
+  private[spark] def asSpec: ModelSpec = {
+    new ModelSpec(
+      name = Some(name),
+      uri = uri.getOrElse(""),
```
Comment (changhiskhan) [inline src/main/scala/ai/eto/rikai/sql/spark/execution/CreateModelCommand.scala:45]: is empty uri a valid model spec?

**Review by changhiskhan — APPROVED** [review]
some nits

File: src/main/scala/ai/eto/rikai/sql/model/ModelSpec.scala (hunk 1 above)
Comment (eddyxu) [reply src/main/scala/ai/eto/rikai/sql/model/ModelSpec.scala:43]: py4j passes a java class, which somehow make it not accessible directly as "ModelSpec.name" and etc.

File: src/main/scala/ai/eto/rikai/sql/spark/execution/CreateModelCommand.scala (hunk 2 above)
Comment (eddyxu) [reply src/main/scala/ai/eto/rikai/sql/spark/execution/CreateModelCommand.scala]: good point, fixed

**Pull request merged**